ios/CredentialsManagerBridge.swift

```diff
@@ -82,9 +82,42 @@ public class CredentialsManagerBridge: NSObject {
         resolve(credentialsManager.clear())
     }
     
-    @objc public func enableLocalAuthentication(title: String?, cancelTitle: String?, fallbackTitle: String?) {
+    @objc public func enableLocalAuthentication(title: String?, cancelTitle: String?, fallbackTitle: String?, evaluationPolicy: NSNumber?) {
         let titleValue = title ?? "Please authenticate to continue"
-        self.credentialsManager.enableBiometrics(withTitle: titleValue, cancelTitle: cancelTitle, fallbackTitle: fallbackTitle)
+        let policyValue = self.convertPolicyInt(policyInt: evaluationPolicy);
+        self.credentialsManager.enableBiometrics(withTitle: titleValue, cancelTitle: cancelTitle, fallbackTitle: fallbackTitle, evaluationPolicy: policyValue as LAPolicy)
+    }
+
+    func convertPolicyInt(policyInt: NSNumber?) -> LAPolicy {
+        if(policyInt == nil || policyInt == 1) {
+            return LAPolicy.deviceOwnerAuthenticationWithBiometrics;
+        }
+        
+        if(policyInt == 2) {
+            return LAPolicy.deviceOwnerAuthentication;
+        }
+        
+        #if os(macOS)
```

Comment:
This SDK does not support macOS. Therefore `deviceOwnerAuthenticationWithWatch` should not be supported.

Reply:
Neither should `deviceOwnerAuthenticationWithBiometricsOrWatch`.

Reply:
See https://developer.apple.com/documentation/localauthentication/lapolicy/deviceownerauthenticationwithwatch and https://developer.apple.com/documentation/localauthentication/lapolicy/deviceownerauthenticationwithbiometricsorwatch.